src/components/ addNewTask/index.js

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react'
```

Comment:
1) why do you need this component?
2) do not use spaces it the name of files or directories

Reply:
So I can remove this component? Because I added a new one.

Reply:
yes